src/util.rs

```diff
@@ -54,6 +54,14 @@ fn show_json<S: Into<String>>(payload: S) {
     }
 }
 
+fn print_error(r: reqwest::StatusCode) {
```

Comment:
this determines the return code as much as printing the error. 

```suggestion
fn exit_with_code(r: reqwest::StatusCode) {
```